transit_odp/site_admin/exports.py

```diff
@@ -144,7 +144,7 @@ def latest_matching_url(dataset) -> str:
 
 
 def overall_avl_timetables_matching(dataset) -> str:
-    if dataset.status != INACTIVE:
+    if dataset.dataset_type == DatasetType.AVL.value and dataset.status != INACTIVE:
```

Comment:
It's not wrong to use `DatasetType.AVL.value`, but `AVLType` is more succinct and that's what it's there for!